File: app/src/main/java/xing/test/mywidget/appwidget/AppInfo.java
package xing.test.mywidget.appwidget;

import android.graphics.drawable.Drawable;

public class AppInfo {
    public AppInfo(String appName, String packageName, Drawable appIcon) {
        this.appName = appName;
        this.packageName = packageName;
        this.appIcon = appIcon;
    }

    public String appName;
    public String packageName;
    public Drawable appIcon;
    public boolean enabled;


    @Override
    public String toString() {
        return "appName = " + appName + "," +
                "packageName = " + packageName + "," +
                "enable = " + enabled;
    }
}

File: app/src/main/java/xing/appwidget/bean/AppInfo.kt
package xing.appwidget.bean

import android.graphics.drawable.Drawable

data class AppInfo(var appName: String, var packageName: String, var appIcon: Drawable, var enabled: Boolean) {

    override fun toString(): String {
        return "appName = " + appName + "," +
                "packageName = " + packageName + "," +
                "enable = " + enabled
    }

}